### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by mgrzeschik** [mvp/uhubctl#465] [General]
Is there any next release planned? Since the last feature to use the disable sysfs option on linux, it would be great to find this option
in the common linux distributions.

**Comment by mvp** [comment]
Thanks for the reminder @mgrzeschik! I am going to do few things and will cut the 2.5.0 release shortly!

**Reply by mvp** [reply]
Just released uhubctl version 2.5.0.
@gfa, can you please release it to Debian, so it can be picked by its downstreams like Ubuntu?
Thanks!
